src/components/MoneyReportHeader.tsx

```diff
@@ -1589,6 +1616,10 @@ function MoneyReportHeader({
                                 primaryAction={primaryAction}
                                 applicableSecondaryActions={applicableSecondaryActions}
                                 ref={kycWallRef}
+                                headerText={isApprovalOptionPressed ? translate('iou.confirmApprovalWithHeldAmount') : ''}
+                                shouldPutHeaderTextAfterBackButton
+                                shouldAlwaysShowHeaderText
```

Comment:
What is the use of this prop? Can we make use of `headerText` instead of this prop but reset `headertext` when we have to hide it.